pkg/capabilities/consensus/ocr3/reporting_plugin.go

```diff
@@ -239,23 +245,35 @@ func (r *reportingPlugin) Reports(seqNr uint64, outcome ocr3types.Outcome) ([]oc
 }
 
 func (r *reportingPlugin) ShouldAcceptAttestedReport(ctx context.Context, seqNr uint64, rwi ocr3types.ReportWithInfo[[]byte]) (bool, error) {
-	id := &pbtypes.Id{}
-	err := proto.Unmarshal(rwi.Info, id)
+	info := &pbtypes.ReportInfo{}
+	err := proto.Unmarshal(rwi.Info, info)
+	if err != nil {
+		r.lggr.Error("could not unmarshal info")
+		return false, err
+	}
+
+	resp := map[string]any{}
+	if info.ShouldReport {
+		resp["report"] = []byte(rwi.Report)
+	} else {
+		resp["report"] = nil
```

Comment:
Maybe we should keep the report for observability purposes? If not then why even generate it (see my earlier comment)...